src/db/models/json/attachment.ts

```diff
@@ -13,16 +13,23 @@ const COST_ATTACHMENT_VALUE = t.intersection([
   t.exact(
     t.partial({
       /**
-       * When necessary, a cost breakdown can be provided with respect to a
-       * particular collection of objects.
-       *
-       * For example, when the object of this attachment is a governing entity,
-       * a breakdown needs to be provided for each of the global clusters,
-       * (even if the governing entity has 0 global clusters)
-       * and the total sum of the breakdown must match the overall cost when
-       * non-empty.
+       * When necessary, a cost breakdown can be provided for each of the global
+       * clusters (even if the governing entity has 0 global clusters), and the
+       * total sum of the breakdown must match the overall cost when non-empty.
        */
-      breakdown: t.array(
+      breakdownByGlobalCluster: t.array(
+        t.exact(
+          t.type({
+            objectId: t.number,
+            cost: t.number,
+          })
+        )
+      ),
+      /**
+       * Unlike global cluster breakdown, service modality breakdown
+       * doesn't need to add up to the total cost
+       */
+      breakdownByServiceModality: t.array(
```

Comment:
I believe it would be better to split this change into two separate commits. One commit should add the `breakdownByServiceModality`, while the other should change the `breakdown` to `breakdownByGlobalCluster`.
Also what about creating common codec for breakdown, and use it